spec/duration.html

```diff
@@ -1107,11 +1110,12 @@ <h1>AddDuration ( _y1_, _mon1_, _w1_, _d1_, _h1_, _min1_, _s1_, _ms1_, _mus1_, _
           1. Let _dateDuration1_ be ? CreateTemporalDuration(_y1_, _mon1_, _w1_, _d1_, 0, 0, 0, 0, 0, 0).
           1. Let _dateDuration2_ be ? CreateTemporalDuration(_y2_, _mon2_, _w2_, _d2_, 0, 0, 0, 0, 0, 0).
           1. Let _dateAdd_ be ? GetMethod(_calendar_, *"dateAdd"*).
-          1. Let _options_ be ! OrdinaryObjectCreate(%Object.prototype%).
-          1. Let _intermediate_ be ? Call(_dateAdd_, _calendar_, « _datePart_, _dateDuration1_, _options_ »).
-          1. Let _end_ be ? Call(_dateAdd_, _calendar_, « _intermediate_, _dateDuration2_, _options_ »).
+          1. Let _firstAddOptions_ be ! OrdinaryObjectCreate(*null*).
+          1. Let _intermediate_ be ? Call(_dateAdd_, _calendar_, « _datePart_, _dateDuration1_, _firstAddOptions_ »).
+          1. Let _secondAddOptions_ be ! OrdinaryObjectCreate(*null*).
+          1. Let _end_ be ? Call(_dateAdd_, _calendar_, « _intermediate_, _dateDuration2_, _secondAddOptions_ »).
```

Comment:
When this was previously changed to cache the GetMethod, I believe I reviewed it but I didn't realize then that this comment applies here too:
> No more `if (!ES.IsTemporalDate(result)) throw new TypeError('invalid result')`; aka `Perform ? RequireInternalSlot(addedDate, [[InitializedTemporalDate]])`. (not good?)